Eatery/Onboarding/Controllers/OnboardingLoginViewController.swift

```diff
@@ -0,0 +1,254 @@
+//
+//  OnboardingLoginViewController.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 11/12/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+import NVActivityIndicatorView
+
+class OnboardingLoginViewController: OnboardingViewController {
+
+    private var loginStackView: UIStackView!
+
+    private var netidPrompt: UILabel!
+    private var netidTextField: UITextField!
+
+    private var passwordPrompt: UILabel!
+    private var passwordTextField: UITextField!
+
+    private var loginButton: UIButton!
+    private var skipButton: UIButton!
+    private var privacyStatementButton: UIButton!
+
+    private var activityIndicator: NVActivityIndicatorView!
+    
+    private var errorView: BRBLoginErrorView!
+    var errorDescription: String? {
```

Comment:
Get rid of this? Can directly assign to `errorView.errorLabel.text` where necessary